subprojects/workers/src/integTest/groovy/org/gradle/workers/internal/WorkerExecutorIntegrationTest.groovy

```diff
@@ -694,6 +694,110 @@ class WorkerExecutorIntegrationTest extends AbstractWorkerExecutorIntegrationTes
         isolationMode << ISOLATION_MODES
     }
 
+    def "workers which use project outputs do not break clean"() {
```

Comment:
Take a look at "can find resources in the classpath via the context classloader using" in this class. It looks like we've been down this road before and that test looks similar to this one, but it's disabled on Windows.